recipes/ncl/build.sh

```diff
@@ -0,0 +1,65 @@
+#!/bin/sh
+
+export CC=${PREFIX}/bin/gcc
+export CXXFLAGS="-fPIC"
+export LDFLAGS="-L${PREFIX}/lib"
+export CPPFLAGS="-I${PREFIX}/include"
+export CFLAGS="-I${PREFIX}/include"
+
+if [ "$(uname)" = "Darwin" ]; then
+    if [ -d "/opt/X11" ]; then
+        x11_lib="-L/opt/X11/lib"
+        x11_inc="-I/opt/X11/include -I/opt/X11/include/freetype2"
```

Comment:
We have `freetype2` in `conda-forge` conda you try to use that instead of the system version?

Also, you may need to add some X11 stuff in the `yum_requirements.txt` I mentioned below.